src/DssSpellCollateral.sol

```diff
@@ -27,11 +66,176 @@ contract DssSpellCollateralAction {
     // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
     //
     // A table of rates can be found at
-    //    https://ipfs.io/ipfs/QmX2QMoM1SZq2XMoTbMak8pZP86Y2icpgPAKDjQg4r4YHn
+    //    https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
```

Comment:
it doesn't look to point to latest on master, you might want to merge latest and resolve conflicts.

Reply:
Let's wait until the latest spell will be executed. Then I will merge the upcoming spell branch.

Reply:
you can specify previous spell in `config.sol`, under `previous_spell` (see [here](https://github.com/makerdao/spells-mainnet/blob/master/src/test/config.sol#L103))

Reply:
Won't work